xss_test.go

```diff
@@ -114,10 +110,53 @@ func TestXSSDriver(t *testing.T) {
 	}
 
 	for _, fi := range dir {
+		p := filepath.Join(baseDir, fi.Name())
+		data := readTestData(p)
 		if strings.Contains(fi.Name(), "-html5-") {
-			runXSSTest(baseDir+fi.Name(), html5)
+			t.Run("XSS Driver - html5", func(t *testing.T) {
```

Comment:
I think the idea was to use the `fi.Name()` in t.Run? otherwise all subcases will be named `XSS Driver - html5` which does not help to know which one failed.

Reply:
My bad, fixed